src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -24,35 +25,69 @@ import {
   E_BENEFITS_URL
 } from './constants';
 
-/*
- * Flatten nested array form data into sibling properties
- *
- * @param {object} data - Form data for a full form, including nested array properties
- */
-export function flatten(data) {
-  const siblings = ['treatments', 'privateRecordReleases', 'privateRecords', 'additionalDocuments'];
-  const formData = cloneDeep(data);
-  formData.disabilities.forEach((disability, idx) => {
-    siblings.forEach(sibling => {
-      if (disability[sibling]) {
-        formData[sibling] = [];
-        formData[sibling][idx] = disability[sibling];
-        delete disability[sibling]; // eslint-disable-line no-param-reassign
-      }
-    });
+/**
+ * Inspects an array of objects, and attempts to aggregate subarrays at a given property
+ * of each object into one array
+ * @param {array} dataArray array of objects to inspect
+ * @param {string} property the property to inspect in each array item
+ * @returns {array} a list of aggregated items pulled from different arrays
+*/
+const aggregate = (dataArray, property) => {
+  const masterList = [];
+  dataArray.forEach(item => {
+
+    if (item[property]) {
+      item[property].forEach(listItem => masterList.push(listItem));
+    }
   });
-  return formData;
-}
 
+  return masterList;
+};
+
+// Moves phone & email out of the phoneEmailCard up one level into `formData.veteran`
+const setPhoneEmailPaths = (veteran) => {
+  const newVeteran = cloneDeep(veteran);
+  const { primaryPhone, emailAddress } = newVeteran.phoneEmailCard;
```

Comment:
You know...that's exactly what view fields are for isn't it? Good idea. I think I'm going to pull this off in a separate PR since it requires changes across prefill, pre-submit, and schema / config files. For now I'll leave this here and nuke it in the next PR.